src/main/core-options.js

```diff
@@ -85,7 +85,7 @@ const options = {
     since: "0.0.10",
     category: CATEGORY_GLOBAL,
     type: "choice",
-    default: "babylon",
+    default: undefined,
```

Comment:
For the support info, you need to specify that the default varies across versions; see https://github.com/prettier/prettier/pull/4426#discussion_r186282273